docs/usage.md

```diff
@@ -424,7 +423,7 @@ If you have any questions or issues please send us a message on [Slack](https://
 
 ## AWS Batch specific parameters
```

Comment:
I think we can remove all the specific params from the docs in the pipeline docs and have it all explained in the awsbatch profile docs in the nf-core/configs repo

Reply:
Ok, this I definitely think we need here if it's an option that is listed and can be provided to the pipeline. Happy to also add it to `nf-core/configs` too.

Reply:
OK for now, as we don't have any docs from the configs on the website, but I do think it should be there.